corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -595,6 +595,20 @@ def get_xml(self):
         del XFormInstance._unsafe_get_xml
 
 
+@contextmanager
+def patch_kafka():
+    def drop_change(self, topic, change_meta):
+        log.debug("not publishing doc_id=%s to %s", change_meta.document_id, topic)
```

Comment:
nit: might be useful to have the word "kafka" in here

Reply:
https://github.com/dimagi/commcare-hq/pull/26768/commits/f24d48191d887eedf808c12389cef25097bf7b10